src/System.Linq.Expressions/src/System/Linq/Expressions/Interpreter/Utilities.cs

```diff
@@ -144,8 +144,8 @@ internal static object GetPrimitiveDefaultValue(Type type)
                     return default(float);
                 case TypeCode.Double:
                     return default(double);
-                //case TypeCode.DBNull:
-                //    return default(DBNull);
+                case TypeCode.DBNull:
+                    return default(DBNull);
```

Comment:
Good point about it not adding any value. Maybe someone thought it was a struct when writing the code initially? I'll change this PR to remove it.

For `TypeExtensions`, you should be fine to change anything except for `GetTypeCode` which is going back to calling `Type.GetTypeCode`. We don't need to reorder anything in that method because it's going away.